js/tools/kermit.js

```diff
@@ -444,16 +442,19 @@ function _pretty_print_table (elements) {
         let spaces = ' '.repeat(width - column.length);
         return str + column + spaces + ' ';
     }, '');
-    print(header_str);
-    elements.forEach((element) => {
+
+    let rows = elements.map((element) => {
         let row = columns.reduce((str, column, i) => {
             let width = column_widths[i];
             let value = element[column].toString();
             let spaces = ' '.repeat(width - value.length);
             return str + value + spaces + ' ';
         }, '');
-        print(row);
+        return row;
     });
+
+    print(header_str);
+    rows.forEach(row => print(row));
```

Comment:
I thought so too, but it breaks since `forEach` passes a bunch of other args into its callback:

```gjs> x = ['foo', 'bar']
foo,bar
gjs> x.forEach(print)
foo 0 foo,bar
bar 1 foo,bar```